app/controllers/automated_tests_controller.rb

```diff
@@ -91,8 +91,17 @@ def populate_autotest_manager
         { key: "#{file}/" }
       else
         file_keys << file
-        { key: file, size: 1,
-          url: download_file_assignment_automated_tests_url(assignment_id: assignment.id, file_name: file) }
+
+        if files_dir.join(file).exist?
+          date = File.mtime(files_dir.join(file))
+          local_date = date.in_time_zone(current_user.time_zone)
+          ""
+          { key: file, size: 1, submitted_date: local_date.strftime("%A, %B, %d, %Y, %I:%M:%S %p %Z"),
```

Comment:
Good research here, but you should use the `I18n.l` (that's a lower case L) localization method to render dates, as different locales render dates differently. You can read more about this here: https://guides.rubyonrails.org/i18n.html#adding-date-time-formats